src/commands/project/init.ts

```diff
@@ -64,9 +53,25 @@ export class ProjectInitCommand extends Command {
       );
     }
 
+    let fslDir: string | undefined = await input({
+      message:
+        "What directory would you like to store your fsl files in? (defaults to current directory)",
```

Comment:
I'd add an option to create the folder if it doesn't exist already (maybe with some prompt like "directory <dir> doesn't exist, create? Y/n")